src/async-flat-map.mjs

```diff
@@ -1,11 +1,11 @@
 import map from './async-map'
 import { asyncIterableCurry } from './internal/async-iterable'
 
-async function * asyncFlatMap (func, iterable) {
+async function * asyncFlatMap (concurrency = 1, func, iterable) {
```

Comment:
This pattern is wrong -- a positional parameter default is used only if arguments.length is shorter than the position of the parameter.

Reply:
A default argument is used if you pass undefined. Example:
```js
const add = (a = 1, b) => a + b
add(3, 2) // 5
add(undefined, 2) // 3
```
asyncIterableCurry deals with passing undefined in the right place

Reply:
Just tested, you're right. The only distinction then is what happens if the user explicitly passes null, which seems less important when they can simply omit the argument.